dxr/static/css/dxr.css

```diff
@@ -332,7 +332,7 @@ mark {
     transform: rotate3d(0, 0, 1, 270deg);
 }
 .content {
-    margin-top: 55px;
+    margin: 55px 0 25px;
```

Comment:
Yes, I meant to do that. Left and right margins were already 0 due to the body's margins being explicitly set to 0. If you'd rather it not be explicit on this element, I'd be happy to add the margin-top declaration back in and change my added line to just be margin-bottom.